library/src/main/java/com/leinardi/android/speeddial/UiUtils.java

```diff
@@ -83,7 +84,15 @@ static int pxToDp(float px) {
      * @param view view to animate.
      */
     public static void fadeOutAnim(final View view) {
-        ViewCompat.animate(view).cancel();
+        if (BuildConfig.DEBUG) {
+            Log.d("tt", "d");
+        } else
+
+        {
+            Log.d("tt", "d");
+        }
+        ViewCompat.animate(view).cancel
+                ();
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.whitespace.MethodParamPadCheck
**Severity**: ERROR
**File**: library/src/main/java/com/leinardi/android/speeddial/UiUtils.java L95

'(' should be on the previous line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-754076342>*